src/core/service.c

```diff
@@ -4309,29 +4309,29 @@ static bool service_notify_message_authorized(Service *s, pid_t pid, FDSet *fds)
         NotifyAccess notify_access = service_get_notify_access(s);
 
         if (notify_access == NOTIFY_NONE) {
-                log_unit_warning(UNIT(s), "Got notification message from PID "PID_FMT", but reception is disabled.", pid);
+                log_unit_debug(UNIT(s), "Got notification message from PID "PID_FMT", but reception is disabled.", pid);
```

Comment:
i'd keep this one. i.e. if we get a notification if notifications are entirely turned off this would be really weird, because we don't even set $NOTIFY_SOCKET in this case, hence if we get a message anyway, things are *really* weird.